proof/crefine/ARM/VSpace_C.thy

```diff
@@ -2233,9 +2234,7 @@ lemma unmapPage_ccorres:
           \<comment> \<open>ARMSection\<close>
           apply (rule ccorres_Cond_rhs)
            apply (rule ccorres_rhs_assoc)+
-           \<comment> \<open>FIXME: The second csymbr rewrites the return relation of the goal.
-                      If this was avoided then folding the dc_def could be removed\<close>
-           apply (csymbr, csymbr, fold dc_def)
+           apply (csymbr, csymbr)
```

Comment:
This was an unexpected but nice benefit of this change.

Reply:
cool